src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts

```diff
@@ -81,15 +83,6 @@ describe('CopySessionModalComponent', () => {
     timeZone: 'UTC',
   };
 
-  it('should snap with some session and courses candidates', () => {
-    component.newFeedbackSessionName = feedbackSessionToCopy.feedbackSessionName;
-    component.courseCandidates = [courseSessionIn, courseCopyTo];
-    component.sessionToCopyCourseId = courseSessionIn.courseId;
-    fixture.detectChanges();
-
-    expect(fixture).toMatchSnapshot();
-  });
-
```

Comment:
let's keep this snapshot test here

Reply:
Thanks for the comment and reply, I actually struggled dealing with running errors with this part. I did not edit any part of “this should snap with some session and course candidates‘, however, it met this error when I kept the snapshot test. Thanks for your support in advance.
![image](https://github.com/TEAMMATES/teammates/assets/132905698/04067fe6-9845-46d5-83c9-7f9a527111f1)

Reply:
hey @Astrid641 it appears that `copy-session-modal.component.spec.ts.snap` was changed, do revert the change and the test should pass

Reply:
Thanks for your reply! I have reverted the change and now all checks have passed, hope all goes well now.